scenario_player/runner.py

```diff
@@ -404,7 +392,28 @@ def _initialize_nodes(
                 tx_hash = resp.json()["tx_hash"]
                 fund_tx.add(tx_hash)
 
-        node_starter = self.node_controller.start(wait=False)
+        try:
+            for node_runner in self.node_controller._node_runners:
+                node_runner.start()
+        except Exception:
+            log.error("failed to start", exc_info=True)
+            raise
+
+        def wait():
+            with gevent.Timeout(100):
```

Comment:
Maybe pull this into a constant for more visibility (I guess right now we're fast enough, but we could run into this)